.github/workflows/ci.yml

```diff
@@ -43,3 +43,43 @@ jobs:
           script: |
             brew install parallel
             parallel --retries 3 ::: "./gradlew connectedCheck"
+  size-report:
+    name: Diffuse report
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: adopt
+        cache: 'gradle'
+    - name: Build sdk for PR and main branch
+      run: |
+        ./gradlew build --stacktrace
+        cp sdk/build/outputs/aar/sdk-release.aar sdk-pr.aar
+        git checkout origin/main
+        ./gradlew build --stacktrace
+        cp sdk/build/outputs/aar/sdk-release.aar sdk-main.aar
+    - id: diffuse
+      uses: usefulness/diffuse-action@v1
+      with:
+        old-file-path: sdk-main.aar
+        new-file-path: sdk-pr.aar
+    - uses: peter-evans/find-comment@v1
+      id: find_comment
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        body-includes: Diffuse report
+    - uses: peter-evans/create-or-update-comment@v1
+      if: ${{ steps.diffuse.outputs.diff-raw != null || steps.find_comment.outputs.comment-id != null }}
+      with:
+        body: |
+          Diffuse report:
+
+          ${{ steps.diffuse.outputs.diff-gh-comment }}
+        edit-mode: replace
+        comment-id: ${{ steps.find_comment.outputs.comment-id }}
+        issue-number: ${{ github.event.pull_request.number }}
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I guess it only needs perms to post a comment

Reply:
GITHUB_TOKEN is defauld token which is always defined, the problem in 'trigger event' we need to add `pull_request` event

I will update PR soon